test/fops/Makefile.am

```diff
@@ -2,7 +2,7 @@ AM_CPPFLAGS = -I$(top_builddir) -I$(top_srcdir)/src/include
 
 noinst_PROGRAMS = t
 t_LDADD = $(top_builddir)/libwiredtiger.la
-t_SOURCES = thread.h file.c fops.c t.c
+t_SOURCES = thread.h file.c fops.c t.c ../utility/util.c
```

Comment:
This list shouldn't have relative paths in it. The right thing to do is to add a new ```Makefile.am``` to the ```test/utility``` directory that references the new source file.